sonar-project.properties

```diff
@@ -0,0 +1,17 @@
+# Github organization linked to sonarcloud
+sonar.organization==testcontainers
```

Comment:
Oh what a subtle mistake... 🙈

Can you fix it in another PR?

Reply:
Yes, #1607 is opened